<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1020</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="title_label">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="mini_mode_button">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="minimize_button">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="maximize_button">
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="close_button">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QListWidget" name="left_sidebar_listwidget">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="stack_scrollarea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>812</width>
               <height>518</height>
              </rect>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="below_bar_frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>69</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QSlider" name="progress_slider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <property name="spacing">
                  <number>13</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="cover_pic_label">
                   <property name="minimumSize">
                    <size>
                     <width>55</width>
                     <height>55</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>55</width>
                     <height>55</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QLabel" name="song_title">
                     <property name="minimumSize">
                      <size>
                       <width>130</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <family>微软雅黑</family>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>TextLabel</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QCheckBox" name="is_like_checkBox">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="minimumSize">
                        <size>
                         <width>25</width>
                         <height>25</height>
                        </size>
                       </property>
                       <property name="sizeIncrement">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="baseSize">
                        <size>
                         <width>0</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>16</width>
                         <height>16</height>
                        </size>
                       </property>
                       <property name="autoRepeatDelay">
                        <number>300</number>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="more_button">
                       <property name="minimumSize">
                        <size>
                         <width>25</width>
                         <height>25</height>
                        </size>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="last_song_button">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pause_button">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="next_song_button">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="progress_labe">
                   <property name="minimumSize">
                    <size>
                     <width>71</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <family>微软雅黑</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                   </property>
                   <property name="text">
                    <string>00:00</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <item>
                    <widget class="QPushButton" name="volume_button">
                     <property name="minimumSize">
                      <size>
                       <width>25</width>
                       <height>25</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="stream_mode_button">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="is_lyrics_checkbox">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="detail_stream_list_button">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
